interfaces/builtin/desktop_launch_test.go

```diff
@@ -101,3 +101,27 @@ func (s *desktopLaunchSuite) TestStaticInfo(c *C) {
 	c.Assert(si.BaseDeclarationPlugs, testutil.Contains, "deny-auto-connection: true")
 	c.Assert(si.BaseDeclarationPlugs, testutil.Contains, "allow-installation: false")
 }
+
+func (s *desktopLaunchSuite) TestDesktopLaunchAndDesktopLegacy(c *C) {
+	const desktopLaunchConsumerYamlWithLaunchAndLegacy = `
+name: other
+version: 0
+apps:
+ app:
+    command: foo
+    plugs: [desktop-launch, desktop-legacy]
+`
+
+	plug, _ := MockConnectedPlug(c, desktopLaunchConsumerYamlWithLaunchAndLegacy, nil, "desktop-launch")
+	slot, _ := MockConnectedSlot(c, desktopLaunchCoreYaml, nil, "desktop-launch")
```

Comment:
desktop-launch slot is already connected in `desktopLaunchSuite.SetUpTest`, and can be accessed through `s.slot`. this line can be removed.

Reply:
Good point both changes. Done.